docs/model_zoo.md

```diff
@@ -18,7 +18,8 @@ To propose a model for inclusion, please submit a [pull request](https://github.
 | Faster RCNN | Object Detection | COCO | 148 MB | [.mar](https://torchserve.pytorch.org/mar_files/fastrcnn.mar) | [persons.jpg](https://github.com/pytorch/serve/blob/master/examples/object_detector/persons.jpg) |Eager|
 | MASK RCNN | Object Detection | COCO | 158 MB | [.mar](https://torchserve.pytorch.org/mar_files/maskrcnn.mar) | [persons.jpg](https://github.com/pytorch/serve/blob/master/examples/object_detector/persons.jpg) |Eager|
 | Text classifier | Text Classification | AG_NEWS | 169 MB | [.mar](https://torchserve.pytorch.org/mar_files/my_text_classifier_v2.mar) | [sample_text.txt](https://github.com/pytorch/serve/blob/master/examples/text_classification/sample_text.txt) |Eager|
-| FCN Resenet 101 | Image Segmentation | COCO | 193 MB | [.mar](https://torchserve.pytorch.org/mar_files/fcn_resnet_101.mar) | [persons.jpg](https://github.com/pytorch/serve/blob/master/examples/image_segmenter/fcn/persons.jpg) |Eager|
+| FCN Resenet 101 | Image Segmentation | COCO | 193 MB | [.mar](https://torchserve.pytorch.org/mar_files/fcn_resnet_101.mar) | [persons.jpg](https://github.com/pytorch/serve/blob/master/examples/image_segmenter/persons.jpg) |Eager|
+| DeepLabV3+ Resenet 101 | Image Segmentation | COCO | 227 MB | [.mar](https://torchserve.pytorch.org/mar_files/deeplabv3_resnet_101.mar) | [persons.jpg](https://github.com/pytorch/serve/blob/master/examples/image_segmenter/persons.jpg) |Eager|
```

Comment:
Update model's name and size as the PyTorch model is not DeepLabV3+ but DeepLabV3 and the size has changed since the latest MAR generation as this size does not correspond to a valid MAR.

Reply:
8e69f32